source/process/release-process.md

```diff
@@ -184,10 +184,13 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit [PRs](https://github.com/tommyvn/mattermost-heroku/commit/94f7c5c0c5d7d2672fb6d62b6a560b4b5c1b5131) to update install version number
     - Close GitHub meta ticket for the release
     - Finalize docs
+      - If reviews are not complete, hold a 30 minute doc review meeting with PMs
```

Comment:
Propose having a doc meeting for anyone who changed or reviewed docs during the release. 

My sense is 30 minutes with everyone together discussing what's changed in docs is the fastest way to find blindspots and will save mana overall for a release.